packages/blockchain-gateway-v1/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Maybe for our tsconfigs, we could extend a base config.
All the settings should be the same across packages, however the includes/excludes may be slightly different.

(I also checked but we can't do this for gitignores unfortunately)